### DISCUSSION TITLE
'none type' object has no attribute 'metric depth' - error

### DISCUSSION THREAD

**Discussion by diverswan9** [nagadomi/nunif#159] [Q&A] [answered]
I keep getting this error - '''none type' object has no attribute 'metric depth''' when trying to process video from "iw3_export.yml". I'm on a PC. Maybe something within my overall PC 'environment' shouldn't or should be there.  - Thanks! I have another PC that I'll try and see if the problem crops up there as well. --- Nope, same thing on my other one...

**Comment by nagadomi** [comment]
It is a bug on my part. I fixed it now.

**Reply by diverswan9** [reply, marked as answer]
> It is a bug on my part. I fixed it now.

Yup! Thanks so much. Now I can move on to test it out.
